src/scriptdev2/scripts/outland/hellfire_citadel/hellfire_ramparts/boss_watchkeeper_gargolmar.cpp

```diff
@@ -116,16 +116,13 @@ struct boss_watchkeeper_gargolmarAI : public ScriptedAI
             m_uiMortalWoundTimer -= uiDiff;
 
         if (m_uiSurgeTimer < uiDiff)
-        {
-            if (Unit* pTarget = m_creature->SelectAttackingTarget(ATTACKING_TARGET_RANDOM, 0))
-            {
-                if (DoCastSpellIfCan(pTarget, SPELL_SURGE) == CAST_OK)
-                {
-                    DoScriptText(SAY_SURGE, m_creature);
-                    m_uiSurgeTimer = urand(12100, 21700);
-                }
-            }
-        }
+            if (const SpellEntry* pSpell = sSpellStore.LookupEntry(SPELL_SURGE))
+                if (Unit* pTarget = m_creature->SelectAttackingTarget(ATTACKING_TARGET_FARTHEST_AWAY, 0, pSpell, SELECT_FLAG_PLAYER))
```

Comment:
instead of pSpell you can use the SPELL_SURGE directly. The core will get the SpellEntry for you.
Also, don't remove the {} from the timers, as it makes the code look bad.